lib/src/voip/backend/mesh_backend.dart

```diff
@@ -658,7 +666,29 @@ class MeshBackend extends CallBackend {
     return;
   }
 
-  Future<void> _onIncomingCall(
+  void _onIncomingCallSetup(GroupCallSession groupCall, CallSession newCall) {
+    if (newCall.room.id != groupCall.room.id ||
```

Comment:
This basically looks like `_onIncomingCallStart`, can you not merge them together and remove the register listener from the replaceCall as well?

Reply:
hm oki